RoosterPlanner.Api/Controllers/ParticipationsController.cs

```diff
@@ -0,0 +1,217 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using RoosterPlanner.Api.Models;
+using RoosterPlanner.Common;
+using RoosterPlanner.Models;
+using RoosterPlanner.Service;
+using RoosterPlanner.Service.DataModels;
+
+namespace RoosterPlanner.Api.Controllers
+{
+    [Authorize]
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ParticipationController : ControllerBase
+    {
+        private readonly ILogger logger;
+        private readonly IParticipationService participationService;
+
+        public ParticipationController(ILogger logger, IParticipationService participationService)
+        {
+            this.logger = logger;
+            this.participationService = participationService;
+        }
+
+        [HttpGet]
```

Comment:
Om je url naar deze methode te definieren hier het pad aangeven: [HttpGet("{personId}")]  =? /api/participations/{personId}  (als Guid)

Reply:
Laten we even mondeling bespreken hoe dat zou werken bij post/patch en bij search methodes